utils/setup/install_deps.sh

```diff
@@ -27,18 +27,18 @@ function set_SUMO_HOME {
 
 function do_install_for_linux {
     echo "Installing sumo (used for traffic simulation and road network)"
-    sudo add-apt-repository "$1" ppa:sumo/stable
+    sudo add-apt-repository $1 ppa:sumo/stable
     sudo apt-get update
-    sudo apt-get install "$1"\
+    sudo apt-get install $1 \
          libspatialindex-dev \
          sumo sumo-tools sumo-doc \
          build-essential cmake
 
     #only a problem for linux
     if ! check_python_version_gte_3_7; then
          echo "A >=3.7 python version not found"
-         if  [[ "$1" = "-y" ]]; then
-              install_python_3_7
+         if  [[ $1 = "-y" ]]; then
```

Comment:
This is fine to do although it is still fine to use `"$1"` as habit so there is not an error when empty in single square bracket comparison.

Reply:
The quotation is what bash script recommends but both work.

Reply:
Reverted to using double quotes for comparing variable to string.